.drone/drone.yml

```diff
@@ -242,7 +242,8 @@ steps:
       - export PATH=$PATH:/usr/local/go/bin
       - GO111MODULE=on go get -u github.com/mitchellh/gox github.com/tcnksm/ghr
       - export PATH="$(go env GOPATH)/bin:$PATH"
-      - make -j4 DOCKER_OPTS="" BUILD_IN_CONTAINER=false RELEASE_BUILD=true RELEASE_TAG=${DRONE_TAG} publish
+      - go mod tidy
```

Comment:
This feels like a bandaid 🤔 I think the problem is that #1767 never got merged to main, so it's missing here and preventing 1.18 builds from working.